scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -424,10 +426,38 @@ def update_motr_hare_keys(self, nodes):
         md_disks_lists = get_md_disks_lists(self, node_info)
         update_to_file(self, self._index_motr_hare, self._url_motr_hare, machine_id, md_disks_lists)
 
+'''
+Write below content to /etc/logrotate.conf file so that mini_mini_provisioner
+log file will be rotated hourly and retained recent max 2 files. Max size of log file is 100M.
+
+Content:
+/etc/cortx/log/motr/<machine-id>/mini_provisioner {
+    hourly
+    size 100M
```

Comment:
why do we want to keep 100M uncompressed log file, any idea how long a 100M file will provide us the data, instead decrease uncompressed size and increase number of compressed file